README.md

```diff
@@ -32,6 +38,7 @@ For Maven:
 </dependency>
 ```
 
+<a name="usage"></a>
```

Comment:
HTML anchor tag is not needed. See [here](https://raw.githubusercontent.com/Vonage/vonage-java-sdk/master/README.md) for example on how to create TOC.

Reply:
Modified